### PR TITLE
Update-Readme

### PR DESCRIPTION
**Pull request by yashjk**

### CODE REVIEW

**Comment by AmanSharma4419** [comment]
Minor Spell Mistake of Each.

**Comment by yashjk** [comment]
> Minor Spell Mistake of Each.

Resolved

**Review by akhildhiman — COMMENTED** [review]
Approved.

**Comment by AmanSharma4419** [comment]
Approved

File: README.md [hunk 1]
```diff
@@ -1 +1,14 @@
-Orbit
+# Exposure System
+
+### Description
+
+Exposure system is a new initiative to expose **Humans of AltCampus** to various different knowledge other than just programming, by providing them with articles and tasks on a regular basis.
```
Comment (prank7) [inline README.md]: @Yjspoder @AmanSharma4419  "various different knowledge" doesn't feel right. Replace it with "various ideas and opinions"

File: README.md [hunk 2]
```diff
@@ -1 +1,14 @@
...
+
+This system also aims at bringing the people out of their comfort zone by giving them challenges, which would be in pairs or in groups. Completion of each of these challenges will result in positive incentives, while not participating will result in disincentives, motivating people to keep carrying on.
```
Comment (prank7) [inline README.md]: @Yjspoder Instead write "As part of motivating people to do stuff the system incentivizes completion of these challenges and punishes people who don't."

File: README.md [hunk 3]
```diff
@@ -1 +1,14 @@
...
+
+### Steps to get started
```
Comment (prank7) [inline README.md:9]: @Yjspoder @AmanSharma4419 Steps should be a bit detailed. For example, you have missed - `npm install`.

**Review by prank7 — CHANGES_REQUESTED** [review]
@Yjspoder  @AmanSharma4419  I have added comments on specific lines. Please fix them.

File: README.md (hunk 1 above)
Comment (yashjk) [reply README.md]: @prank7 Resolved

File: README.md (hunk 2 above)
Comment (yashjk) [reply README.md]: @prank7 Resolved

File: README.md (hunk 3 above)
Comment (yashjk) [reply README.md:9]: @prank7 Resolved

**Pull request merged**